src/redis_request.cc

```diff
@@ -61,13 +71,13 @@ Status Request::Tokenize(evbuffer *input) {
           } catch (std::exception &e) {
             return Status(Status::NotOK, "Protocol error: invalid multibulk length");
           }
+          if (isOnlyLF || multi_bulk_len_ > (int64_t)PROTO_MULTI_MAX_SIZE) {
+            return Status(Status::NotOK, "Protocol error: invalid multibulk length");
+          }
```

Comment:
Why do we disallow `\n` as EOL in the multi-bulk scenario?

Reply:
Redis also disallowed \n in multi bulk protocol, so we just keep consistent with Redis here.

Reply:
Thanks for your explanation. I get the point now.